elements/xml_form_elements.module

```diff
@@ -303,11 +303,35 @@ function xml_form_elements_form_element_tabs_ajax_add(FormElement $element, arra
     if ($element->controls['#type'] == 'tags') {
       foreach ($element->children as $child) {
         $child->default_value = $child->getOriginalDefaultValue();
-        if (empty($child->default_value)) {
+
+        // We should orphan object only if they have no default AND it is not
+        // the last child.  We must retain one, otherwise we'll get AJAX an
+        // AJAX error when adding an element after adding another tab.
+        $no_default = empty($child->default_value);
+        $is_not_last = count($element->children) > 1;
+
+        if ($no_default && $is_not_last) {
           $child->orphan();
         }
       }
+
+      // In order to provide functionality identical to when the tab is first
+      // created, we need to dedeupe based on the first element's default
+      // value.  Otherwise we can wind up with multiples of the same default
+      // value.
```

Comment:
Actually... What exactly is the goal, here? Are we trying to have the same value as when the form originally loaded for input, because this will not really work... Because we have clones of arbitrary members, different tags with different defaults, there's no strong assertion that can be made... Any added will appear to duplicate, any deleted will not appear... Though this is more because of [setting the default above](https://github.com/Islandora/islandora_xml_forms/pull/214/files#diff-9fd1f16ddbdc498ecc545353130d9253L305).

... It might be simplest to avoid setting the default, so it just clones the values from the tab on which the "add" button was pressed?